cmd/krew/cmd/internal/setup_check_test.go

```diff
@@ -43,6 +43,40 @@ func TestIsBinDirInPATH_secondRun(t *testing.T) {
 	}
 }
 
+// Test case when PATH content is not well normalized
+func TestIsBinDirInPATH_NonNormalized(t *testing.T) {
+	tempDir := testutil.NewTempDir(t)
+
+	// backup and restore initial PATH value
+	backupPath := os.Getenv("PATH")
+	defer func(backupedPath string) {
+		err := os.Setenv("PATH", backupedPath)
+		if err != nil {
+			t.Errorf("fail to restore PATH env variable after test")
+		}
+	}(backupPath)
+
+	// given
+	// set PATH with non-normalized folder path
+	err := os.Setenv("PATH", tempDir.Root()+"/home/../home/user//./.krew/bin")
+	if err != nil {
+		t.Fatalf("fail to set PATH with not clean path")
+	}
+	paths := environment.NewPaths(tempDir.Path("/home/../home/user//./.krew"))
```

Comment:
Please extract the path into a const variable in the test function, so that you don't have to repeat it.